docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -43,10 +43,11 @@ $ docker build -t alluxio .
 ```
 
 By default, this will build an image for the latest released version of Alluxio. To build
-from a local Alluxio tarball instead, you can use `--build-arg`
+from a local Alluxio tarball or a different downloadable tarball, you can use `--build-arg`
 
 ```bash
 $ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=http://downloads.alluxio.org/downloads/files/1.4.0/alluxio-1.4.0-bin.tar.gz .
```

Comment:
Could you add comments in the code block, it is not clear that only one of these should be executed.